From 89b8f017e3cc3e63ee220be3e4edb55e49c934ff Mon Sep 17 00:00:00 2001
From: Noam Lessner <nlessner@habana.ai>
Date: Tue, 28 Jan 2020 22:32:55 +0200
Subject: [PATCH] [SW-8228] goya2: base addr is in fw_sram

change base address of ppboot to start of fw_sram address
in arc processor, add 12 bytes because of image header of
secure boot (total image size, version, ppboot size).

start of code will be at offset 16, because linker scripts
defines alignment of 8 bytes.

Change-Id: Id6b86fb3d9d1077064a2180617a28d0e58efbdfa
Signed-off-by: Noam Lessner <nlessner@habana.ai>
---

diff --git a/goya2_arc/common.h b/goya2_arc/common.h
index 9ebee60..7be24fb 100644
--- a/goya2_arc/common.h
+++ b/goya2_arc/common.h
@@ -7,7 +7,7 @@
 
 #include <preboot.h>
 
-#define PPBOOT_TEXT_BASE	0xf0000000
+#define PPBOOT_TEXT_BASE	0x1B05000C
 #define PREBOOT_TEXT_BASE	(PPBOOT_TEXT_BASE + PPBOOT_SIZE)
 
 #define mmCPU_PLL_BASE		0x4A2000
